packages/ui/src/memberships/components/MemberProfile/MemberDetails.tsx

```diff
@@ -44,10 +44,12 @@ export const MemberDetails = React.memo(({ member }: Props) => {
 
   return (
     <SidePaneTable>
-      <SidePaneColumn>
-        <SidePaneLabel text="About" />
-        <SidePaneText>{memberDetails?.about || ''}</SidePaneText>
-      </SidePaneColumn>
+      {memberDetails?.about && (
+        <SidePaneColumn>
+          <SidePaneLabel text="About" />
+          <SidePaneText>{memberDetails?.about}</SidePaneText>
```

Comment:
```suggestion
          <SidePaneText>{memberDetails.about}</SidePaneText>
```

Reply:
The check above (`memberDetails?.about &&`) should tell TypeScript that `.about` is not `undefined`.